build-images.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+TAG=$1
+if [ -z "$TAG" ]; then
+  TAG="local"
+fi
+
+case "$TAG" in
+  "local")
+    TARGET="dev"
+    ;;
+  "prod")
+    TARGET="prod"
+    ;;
+  *)
+    echo "Tag must be 'local' or 'prod'"
+    exit 2
+esac
```

Comment:
Yeah, this was mainly just a quick fix for a simple test that has propagated. It needs correcting in a few places to bring tag and image stage target into sync. I don't think we need a `local` tag, and can just call it `dev`. 

But, this is all because of the stage names we have in `Dockerfile`. We can't just take user input and have it build a non-existent stage. Ideally, we have a nicer script that supports args in a more fancy way for both `build-images.sh` and `helm-install.sh`, as they are both fairly simple and restrictive at the moment. I'd like to be able to do more fancy things, like YAML parsing/injecting (not most straight forward thing to do with sed or in shell itself). 

For instance, one thing I'd like to do, is separate out build target from image tag. We should probably still echo a warning if its not a "pre-defined" tag name used in the override yamls for the helm chart, but at least one could build any stage of the `Dockerfile` and tag it with whatever name they want as well. 

I have been rolling around proposing we look at bringing in a CLI framework to stand on top of the project, and potentially have it call manage.py scripts as well, centralizing onto a cfgov cli. We could even have it where it gets pip installed and becomes a PATH callable executable via [`setup.py` console scripts](https://typer.tiangolo.com/), but this is probably out of scope for this initial foray into Helm.

Reply:
Just did a quick check here are all the `.sh` files we have, which could be consolidated into a cfgov CLI.

```
./setup.sh
./runserver.sh
./docker/deployable-zipfile/build.sh
./docker/deployable-zipfile/docker-entrypoint.sh
./docker/deployable-zipfile/_build.sh
./docker/docs/entrypoint.sh
./show_coverage.sh
./docker-entrypoint.sh
./scripts/check-npm-cache.sh
./backend.sh
./initial-data.sh
./refresh-data.sh
./dump-data.sh
./frontend.sh
./extend-environment.sh
./build-images.sh
./helm-install.sh
```